Playbooks/Update-CVE-IPs-WatchListwithGreyNoise/azuredeploy.json

```diff
@@ -0,0 +1,1301 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "comments": "This playbook uses the GreyNoise API to search for interesting IPs discovered in the last day tagged per each CVE found in the mode you setup.",
+        "author": "Nathan Swift"
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "Update-CVE-IPs-WatchListwithGreyNoise",
+            "type": "string"
+        },
+        "SentinelWorkspaceName": {
+            "defaultValue": "Sentinel Workspace Name",
+            "type": "string"
+        },
+        "SentinelWorkspaceId": {
+            "defaultValue": "Sentinel Workspace Id",
+            "type": "string"
+        },
+        "SentinelWorkspaceRGName": {
+            "defaultValue": "Sentinel Resource Group Name",
+            "type": "string"
+        },
+        "SentinelWorkspaceSubscriptionId": {
+            "defaultValue": "Sentinel Azure Subscription Id",
+            "type": "string"
+        },
+        "GreyNoiseAPIKey": {
+            "type": "securestring"
+        },
+        "CVEMode": {
```

Comment:
Use allowedValues to restrict the CVEMode options. 
https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/parameters#allowed-values